SISRP-8153 - Fluid - add colors red (#ff3037) and green (#0d863d) to …

marcpechaitis opened the pull request:
…css; create class to style errors with input values

https://jira.berkeley.edu/browse/SISRP-8153

marcpechaitis committed:
SISRP-8153 - Fluid - add colors red (#ff3037) and green (#0d863d) to css; create class to style errors with input values

https://jira.berkeley.edu/browse/SISRP-8153

marcpechaitis commented:
@christianv Please review

marcpechaitis committed:
SISRP-8153 - Fluid - add colors red (#ff3037) and green (#0d863d) to css; create class to style errors with input values

https://jira.berkeley.edu/browse/SISRP-8153

Resolve build errors

christianvuerings commented on src/css/cs/items/_kitchensink.scss:
```diff
@@ -312,6 +313,21 @@
+  .uc-input-error {
+    > .ps_box-control input[type="text"] {
+      background-color: $uc-color-red;
+      background-image: none;
```
@marcpechaitis can we remove it?

christianvuerings commented on src/css/cs/items/_kitchensink.scss:
```diff
@@ -312,6 +313,21 @@
+  }
+
+  .psc_error {
+    @media all {
```
@marcpechaitis can we try it without the `@media all`?

marcpechaitis committed:
SISRP-8153 - Fluid - add colors red (#ff3037) and green (#0d863d) to css; create class to style errors with input values

https://jira.berkeley.edu/browse/SISRP-8153

Fixed per feedback from @christianvu

marcpechaitis committed:
SISRP-8153 - Fluid - add colors red (#ff3037) and green (#0d863d) to css; create class to style errors with input values

https://jira.berkeley.edu/browse/SISRP-8153

Resolved build issues

marcpechaitis committed:
SISRP-8153 - Fluid - add colors red (#ff3037) and green (#0d863d) to css; create class to style errors with input values

https://jira.berkeley.edu/browse/SISRP-8153

Removing trailing spaces in _aa_planner.css

marcpechaitis commented:
@christianv r4r
